### ISSUE TITLE
abaplint suppression pseudo comment

### ISSUE THREAD

**Issue by EduardoCopat** [abaplint/abaplint#193]
When running Code Inspector checks, you can add a pseudo comment to suppress false negatives such as `"#EC NEEDED`. Something similar could exist in abaplint. 

Using the example from issue #192, which currently returns a parsing error:

```abap
SELECT SINGLE carrid, connid, fldate FROM sflight
      INTO CORRESPONDING FIELDS OF @return
      WHERE carrid = @i_flight_number.
```

One could add a pseudo-comment after the statement:

```abap
SELECT SINGLE carrid, connid, fldate FROM sflight
      INTO CORRESPONDING FIELDS OF @return
      WHERE carrid = @i_flight_number. "#abaplint suppress
```

**Comment by larshp** [comment]
yeah, I'm just afraid that bugs will not be reported

fixing #192 should be quite easy

**Comment by EduardoCopat** [comment]
> yeah, I'm just afraid that bugs will not be reported

Good point. Thanks!

**Comment by larshp** [comment]
but lets just keep it open, it is a good idea, I'm just lazy 😄

**Comment by EduardoCopat** [comment]
Nasty hack:

`"test": "abaplint 2>&1 > tee abaplintresults"`


```shell
#Remove false positives
sed -i '/Parser error(Unknown statement)/d' ./abaplintresults

if grep -q "abaplint: 0 issue(s) found" abaplintresults; then
  echo "abap lint OK"
  exit 0
else
  cat abaplintresults
  exit 1
fi
```

**Comment by larshp** [comment]
does it remove parser error findings from the output?

**Comment by EduardoCopat** [comment]
Yes...

**Comment by larshp** [comment]
it should work disabling them in the configuration, like https://github.com/sbcgua/abap_data_parser/blob/master/abaplint.json

**Comment by EduardoCopat** [comment]
🤦‍♂️

**Comment by larshp** [comment]
closing, see https://github.com/abaplint/abaplint/issues/695 instead
